\documentclass[11pt, letterpaper]{article}
\usepackage[top=1in, bottom=1in, left=1in, right=1in]{geometry}

% Math, graphics, and bibliography
\usepackage{amsmath,amssymb,amsfonts,mathrsfs,mathtools}

% My preferred fonts
\usepackage[T1]{fontenc}
\usepackage{lmodern}
\usepackage[sc]{mathpazo}
\usepackage{textcomp}

\usepackage{color}
\usepackage{tcolorbox}
\tcbuselibrary{breakable}
\tcbuselibrary{skins}
\usepackage{units}

\usepackage[parfill]{parskip} % For block paragraphs
\definecolor{gray}{gray}{0.4}
\newenvironment{reviewer}{\itshape\color{gray}}{}

\tcbset{skin=enhanced}

\newenvironment{manuscript}[1]{\begin{center}\begin{tcolorbox}[colback=green!5!white,colframe=green!75!black,width=0.8\textwidth,title={#1},breakable,fonttitle=\bfseries]}{\end{tcolorbox}\end{center}}


\begin{document}
\today\\

{\itshape Francis J. Doyle III}\\
Dept.\ of Chemical Engineering\\
Univ.\ of California, Santa Barbara\\
Santa Barbara, CA 93106-5080\\

{\itshape Leah Edelstein-Keshet}\\
Editorial Board Member\\
Biophysical Journal\\

Thank you for your email on September 24th, 2014, inviting us to submit a revised version of our manuscript, ``Amplitude metrics for cellular circadian bioluminescence reporters'' (MS: 2014BIOPHYSJ304278R) by Peter C. St. John, Stephanie R. Taylor, John H. Abel, and Francis J. Doyle~III.

We thank the reviewers for their second round of comments of the manuscript, in particular Reviewer \#2 for their continued attention to detail. We believe addressing these remaining concerns and suggestions have improved the polish of the manuscript.

Please see our detailed responses to the reviewers below: 

\section*{Reviewer \#1}
\begin{reviewer}
The authors have largely responded you my concerns. 
 
Typos and suggestions 
 
- p. 2: "fail to capture the collective dynamics of a population oscillators" 
\end{reviewer}

This typo has been addressed, the sentence now reads:

\begin{manuscript}{Page 2}
  Ordinary differential equation (ODE) models of gene regulation are capable of describing the amplitude and phase-resetting behavior of single cells, but fail to capture the collective dynamics of a population {\bfseries of} oscillators.
\end{manuscript}
 
\begin{reviewer}
- mu is defined a few lines after it is first used in Eq. (20). Perhaps define it first? 
\end{reviewer}

This change has also been made.

\begin{manuscript}{Page 7}
To define an amplitude change metric for such a case, we compare a perturbed trajectory $x(t)$ to a phase-shifted limit cycle reference $y(t)$, for which $x(t) \to y(t)$ for sufficiently long times.
Since $x(t)$ approaches the reference as $t \to \infty$, {\bfseries the means of both trajectories are equal and can be calculated by
\begin{equation}
  \mu \coloneqq \int_0^{2\pi} \frac{x^\gamma(\theta)}{2\pi} \; d\theta.
  \tag{20}
\end{equation}
The amplitude change metric is defined as}
\begin{equation}
  \begin{aligned}
    \Delta A (x(t), y(t)) &\coloneqq \int_0^\infty (x(t) - \mu)^2 - (y(t) - \mu)^2 \; dt\\
    &= \int_0^\infty h(t) \; dt.
  \end{aligned}
  \tag{21}
\end{equation}
\end{manuscript}

\section*{Reviewer \#2}
\begin{reviewer}
The revised manuscript becomes much more clear. The authors addressed most of my concerns in their revision. However, I have still couple concerns regarding the manuscript. 
 
1) In the previous review, I was concerned about using the Gillesplie algorithm for the model with non-elementary reactions because recent studies have clearly showed the inaccuracy of doing that (Thomas, Straube and Grima BMC Syst Biol (2012), Agarwal et al. J Chem Phy (2012) and Kim, Josic and Bennett Biophy J (2014)). Regarding this concern, the author showed that their continuous approximation also works for the model based on mass-action kinetics (Model 2). But, in the manuscript, the approximation is more accurate for Model 2 than other models, which are based on non-elementary reactions. This indicates that the inaccuracy coming from the Gillespie algorithm with non-elementary reaction appears to contribute the error seen in Model 1 and 3. Furthermore, I am mostly concerned that this manuscript can mislead the audience that using Gillespie algorithm with non-elementary reactions is justified. Please, clearly discuss the limitation in this approach. 
\end{reviewer}

\begin{reviewer}
2) Another concern is regarding the major result of the manuscript (Fig. 5). The author perturbed the degradation of Per mRNA parameter in response to light. However, this conflicts with the previous experimental studies, which have shown that light promotes the transcription of Per mRNA via CREB, not degradation. 
\end{reviewer}

\begin{reviewer}
3) The key goal of this manuscript is distinguishing the single-cell level and population-level amplitude change. For this, the authors applied their tool to the experimental data (Fig. 5). However, Fig. 5 does not clearly show how their tool achieves such goal. Please provide additional figures. For instance, $x_{ss}$ (steady-state perturbed trajectory) for perturbed and unperturbed cases, $p(\theta, t)$ for unperturbed case as well as perturbed case etc, so that readers can clearly see the difference of effect via single cell level and population level.  
\end{reviewer}

This is a good suggestion, and we have therefore added a figure to explicitly highlight the contributions from the population and single-cell level.
Due to lack of space in the main text and desire not to further complicate the original figure, we have added this figure as Figure~S4. A reference to the figure was added to the main text of the manuscript:

\begin{manuscript}{Page 16}
The phase probability density function for the light-sensitive model trajectory is shown in Fig.~5C for several representative time points.
\ldots
{\bfseries Amplitude change in this case is mediated both at the population and single-cell level, as indicated by their representative ARCs. The contributions from each source are summarized in Fig.~S4.}
\end{manuscript}

with the following figure and caption added to the supplementary material:

\begin{manuscript}{Supplemental Info, Page 7}
  \begin{center}
    \includegraphics[width=.85\textwidth]{figures/figure_S4.pdf}\\
  \end{center}
{\bfseries Figure S4: Evidence of single-cell and population-level amplitude change.}
({\bfseries A}) The simulated control trajectory as shown in Fig.~5 is plotted again as $\bar{x}(\tilde{t})$.
The simulated light-sensitive trajector is also replotted as $\hat{x}(\tilde{t})$, which incorperates both single-cell and population-level effects.
The difference between this trajectory and the steady-state perturbed trajectory following both pulses, $\hat{x}_{ss}(\tilde{t})$, demonstrates the effect of single-cell level amplitude change in this particular case. 
Single-cell amplitudes are greatly increased following the first pulse, and altered significantly following the second.
({\bfseries B})
Comparison of the population densities for the control (dashed) and perturbed (solid) simulations.
The perturbed population is greatly desynchronized following the first light pulse, as indicated by its flatness relative to the control trajectory. 
Following the second pulse, the perturbed population is more concentrated than the control, indicating higher amplitudes. 
These comparisons demonstrate the changes in population synchrony which occur due to the light pulse. 
\end{manuscript}



\end{document}
